external.bzl

```diff
@@ -272,334 +275,424 @@ def _java_dependencies():
     )
 
 def _go_dependencies():
-    maybe(
-        go_repository,
+    go_repository(
         name = "com_github_golang_protobuf",
         build_file_proto_mode = "disable_global",
-        custom = "protobuf",
         importpath = "github.com/golang/protobuf",
         patch_args = ["-p1"],
-        patches = ["@io_bazel_rules_go//third_party:com_github_golang_protobuf-extras.patch"],
-        tag = "v1.3.1",
+        patches = [
+            "@io_bazel_rules_go//third_party:com_github_golang_protobuf-extras.patch",
+            "@//third_party/go:new_export_license.patch",
```

Comment:
Due to the leading `@` this will pull the patch from the repo that loads kythe in. This would force someone who loads kythe as a dependency to vendor this patch file into the repo. It might be preferable to omit the `@` and just load it from` @io_kythe`.

Here and elsewhere

Reply:
https://github.com/kythe/kythe/pull/4282